test/auto/db_spec.lua

```diff
@@ -546,6 +546,10 @@ describe("sqlite.db", function()
   end)
 
   describe(":select", function()
+    if vim.fn.executable("curl") then
+        pending("'curl' program is not available")
```

Comment:
what's pending 🤔 , part of busted framework?

Reply:
yes. To test you can replace curl with `dadadsadad` to fail the condition and check the output.